setup.py

```diff
@@ -12,44 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Build script for setuptools."""
-import sys
+"""setuptools based stub for the "editable" installations"""
+from setuptools import setup
```

Comment:
```suggestion

from setuptools import setup
```
(PEP8 suggests to have a blank line between the docstring and the import section, e.g. https://peps.python.org/pep-0008/#module-level-dunder-names)